### PR TITLE
Add arm64 arch and NXP ls2080ardb platform

### PR DESCRIPTION
**Pull request by shengzhou**
Note Prerequisites:  
1. to build arm64 arch in ONL, it needs to use new builder8:1.5 
2. it needs to first apply pull request https://github.com/floodlight/infra/pull/54 for submodule infra build
These patches passed build test and can run on nxp arm64 ls2080ardb.

### CODE REVIEW

**Comment by jnealtowns** [comment]
@shengzhou Do you have a qemu target setup for testing these changes under emulation without your RDB?

File: builds/arm64/rootfs/builds/Makefile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+include $(ONL)/make/config.arm64.mk
+
+#
+# Default to include all available powerpc platforms.
```
Comment (jnealtowns) [inline builds/arm64/rootfs/builds/Makefile]: ^powerpc^arm64

File: packages/platforms/nxp/arm64/arm64-nxp-ls2080ardb/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+*x86*64*cel*redstone*xp*.mk
```
Comment (jnealtowns) [inline packages/platforms/nxp/arm64/arm64-nxp-ls2080ardb/.gitignore]: Wrong pattern.

**Comment by shengzhou** [comment]
Updated patches to address jnealtowns' comments.

**Comment by shengzhou** [comment]
@jnealtowns   I have no a qemu target setup for arm64 (it needs to add qemu arm64 in ONIE and ONL), I just verified the patches on NXP arm64 LS2080A RDB platform.

**Pull request closed without merging**